src/Microsoft.DotNet.Darc/src/Darc/Options/VirtualMonoRepo/GetRepoVersionCommandLineOptions.cs

```diff
@@ -0,0 +1,22 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using CommandLine;
+using Microsoft.DotNet.Darc.Operations;
+using Microsoft.DotNet.Darc.Operations.VirtualMonoRepo;
+using Microsoft.Extensions.DependencyInjection;
+using System.Collections.Generic;
+
```

Comment:
Not sure if we want to have nullability enabled everywhere, I guess it's a bit easier if we don't have it here

Reply:
If I have to choose between `null!` and turning nullability off, the former is a clear winner to be honest

Reply:
Will turn on